tests/validation/tests/v3_api/test_multi_cluster_app.py

```diff
@@ -73,27 +76,524 @@
+def test_multi_cluster_role_change():
+    assert_if_valid_cluster_count()
+    targets = []
+    for projectid in project:
+        targets.append({"projectId": projectid, "type": "target"})
+    client = global_client["client"]
+    original_role = ["project-member"]
+    multiclusterapp = client.create_multiClusterApp(templateVersionId=TEMP_VER,
+                                                    targets=targets,
+                                                    roles=original_role,
+                                                    name=random_name(),
+                                                    answers=[answer])
+    multiclusterapp = wait_for_mcapp_to_active(client, multiclusterapp)
+    new_role = ["cluster-owner"]
+    multiclusterapp = client.update(multiclusterapp, roles=new_role)
+    start = time.time()
+    while multiclusterapp['roles'] != new_role:
+        if time.time() - start > 120:
+            raise AssertionError(
+                "Timed out waiting")
+        time.sleep(10)
+        if multiclusterapp['roles'] == new_role:
+            break
+    assert multiclusterapp['roles'] == new_role, "role did not update"
+    delete_multi_cluster_app(multiclusterapp)
+
+
+@pytest.mark.usefixtures("create_project_client")
+def test_multi_cluster_answers():
+    assert_if_valid_cluster_count()
+    targets = []
+    for projectid in project:
+        targets.append({"projectId": projectid, "type": "target"})
+    client = global_client["client"]
+    multiclusterapp = client.create_multiClusterApp(templateVersionId=TEMP_VER,
+                                                    targets=targets,
+                                                    roles=ROLES,
+                                                    name=random_name(),
+                                                    answers=[new_answers])
+    multiclusterapp = wait_for_mcapp_to_active(client, multiclusterapp)
+    hold = multiclusterapp['answers'][0]
+    values = hold["values"]
+    assert str(values) == str(new_answers["values"])
+    delete_multi_cluster_app(multiclusterapp)
+
+
+@pytest.mark.usefixtures("create_project_client")
+def test_multi_cluster_edit_answers():
+    assert_if_valid_cluster_count()
+    targets = []
+    for projectid in project:
+        targets.append({"projectId": projectid, "type": "target"})
+    client = global_client["client"]
+    multiclusterapp = client.create_multiClusterApp(templateVersionId=TEMP_VER,
+                                                    targets=targets,
+                                                    roles=ROLES,
+                                                    name=random_name(),
+                                                    answers=[answer],
+                                                    )
+    multiclusterapp = wait_for_mcapp_to_active(client, multiclusterapp)
+    multiclusterapp = client.update(multiclusterapp, roles=ROLES, answers=[new_answers])
```

Comment:
can avoid declaring global variable "new_answers" and instead change one of the answers (dictionary)value